src/coreclr/jit/emitxarch.cpp

```diff
@@ -9270,7 +9270,7 @@ void emitter::emitDispIns(
  *  Output nBytes bytes of NOP instructions
  */
 
-static BYTE* emitOutputNOP(BYTE* dst, size_t nBytes)
+BYTE* emitter::emitOutputNOP(BYTE* dstRW, size_t nBytes)
```

Comment:
I don't see if we access `writeableOffset` somewhere inside this function or why do we need to make it a member?

Reply:
This should be changed back to static. I were originally passed in the read-execute address and using the writeableOffset inside until I've found the recursive nature of this function. Then I've forgotten to revert this change.